internal/internal.go

```diff
@@ -169,11 +177,15 @@ func (p *envoyExtAuthzGrpcServer) listen() {
 		"enable-reflection": p.cfg.EnableReflection,
 	}).Info("Starting gRPC server.")
 
+	p.manager.UpdatePluginStatus(PluginName, &plugins.Status{State: plugins.StateOK})
+
+	p.server.GetServiceInfo()
```

Comment:
Why is this call being made ?

Reply:
Oops, was just junk leftover from me exploring the API via intellisense. Removed it.